src/EnergyPlus/ZoneTempPredictorCorrector.hh

```diff
@@ -104,6 +104,175 @@ namespace ZoneTempPredictorCorrector {
         Array1D<Real64> ThermalComfortAdaptiveCEN15251_Central;
     };
 
+    struct SumHATOutput
```

Comment:
New struct used for predictor-corrector's sumHAT function output which happens to do things a bit differently than the other `sumHATSurf` function (might need some investigation at some point).